### PR TITLE
chore(readme): migrator readme

### PR DESCRIPTION
**Pull request by dashrews78**
## Description

Update migrator readme with a gorm note.

## Checklist
- [ ] Investigated and inspected CI test results
~- [ ] Unit test and regression tests added~
~- [ ] Evaluated and added CHANGELOG entry if required~
~- [ ] Determined and documented upgrade steps~
~- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs]~(https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### Here I tell how I validated my change

TODO(replace-me)  
Use this space to explain **how you validated** that **your change functions exactly how you expect it**.
Feel free to attach JSON snippets, curl commands, screenshots, etc. Apply a simple benchmark: would the information you
provided convince any reviewer or any external reader that you did enough to validate your change.

It is acceptable to assume trust and keep this section light, e.g. as a bullet-point list.

It is acceptable to skip testing in cases when CI is sufficient, or it's a markdown or code comment change only.  
It is also acceptable to skip testing for changes that are too taxing to test before merging. In such case you are
responsible for the change after it gets merged which includes reverting, fixing, etc. Make sure you validate the change
ASAP after it gets merged or explain in PR when the validation will be performed.  
Explain here why you skipped testing in case you did so.

Have you created automated tests for your change? Explain here which validation activities you did manually and why so.

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

**Review by connorgorman — APPROVED** [review]

File: migrator/README.md [hunk 1]
```diff
@@ -272,8 +275,12 @@ In migrator, there are a multiple ways to access data.
 3. Duplicate the Postgres Store
    This method is used in version 73 and 74 to migrate all tables from RocksDB to Postgres. In addition to frozen schema,
    the store to access the data are also frozen for migration. The migrations with this method are closely associated
-   with current release eg. search/delete with schema and the prototypes of the objects. This method is NOT recommended for
-   4.0 and beyond.
+   with current release eg. search/delete with schema and the prototypes of the objects. 
+
+   Now that stores are built upon a generic store you would need to make a copy of the store and remove
+   references to SAC and such.  We should be careful to not relying on the search framework for queries as those
```
Comment (charmik-redhat) [inline migrator/README.md:281]: Should the copied stores also refer to frozen schemas instead of the latest schemas?

Comment (charmik-redhat) [inline migrator/README.md:281]: nit: careful not to rely on the search framework

File: migrator/README.md [hunk 2]
```diff
@@ -158,6 +158,9 @@ In migrator, there are a multiple ways to access data.
 
 2. Gorm
 
+    Consideration:  If the migration is dealing with multiple tables tied together it will may be simpler for
```
Comment (charmik-redhat) [inline migrator/README.md:161]: nit: it may be simpler

File: migrator/README.md (hunk 1 above)
Comment (dashrews78) [reply migrator/README.md]: done

Comment (dashrews78) [reply migrator/README.md]: excellent point.

**Review by charmik-redhat — APPROVED** [review]

**Pull request merged**